perceval/backends/core/github.py

```diff
@@ -502,6 +522,7 @@ class GitHubClient(HttpClient, RateLimitHandler):
 
     _users = {}       # users cache
     _users_orgs = {}  # users orgs cache
+    _commit_comments = {}  # commit_comments cache
```

Comment:
it is probably better to call this variable `_commit hashes` or `_commits`, since it is used to store hashes. What do you think ?

Reply:
Hey @valeriocos,
As the `_users` is storing the response from the endpoint `https://api.github.com/users/<login>` so that the same request is not issued again (if the same user comes),
similarly `_commit_comments` is storing the response from the endpoint `https://api.github.com/repos/<owner>/<repository>/commits/<commit-hash>/comments`, which gives the comments on that particular commit